content/en/Features/service2service.md

```diff
@@ -1,32 +1,82 @@
 ---
-title: "Service Mesh"
-linkTitle: "Service Mesh"
+title: "Service Networking"
+linkTitle: "Service Networking"
 date: 2022-01-03
 draft: false
 description: >
-  Communication between services
+  Networking aspects for services
 ---
 
-- All inter-communication is handled by the [Linkerd](https://linkerd.io/) service mesh. Linkerd provides mTLS, load balancing, retries and many other reliability/security features.
+## Overview
+One of the main subjects Opta answers for its users is service networking -- how services talk to one another
+and the external communication. Opta achieves this by utilizing a service mesh, kubernetes ingress, and kubernetes
+service discovery, as detailed below.
 
-- Opta creates a private subnet where all the services run. So every service is inaccessible from the outside, by default. You can enable external requests by using the `public_uri` [field](/reference/aws/service_modules/aws-k8s-service).
+## Service Mesh
+All inter-service communication is handled by the [Linkerd](https://linkerd.io/) service mesh, which Opta installs by default.
+Linkerd provides out-of-the-box mTLS, load balancing, retries and many other 
+[reliability/security/qos features](https://linkerd.io/2.11/features/). Opta's `k8s-service` modules are designed to
+seamlessly assimilate into the installed service mesh. Combined with Linkerd's official principles of 
+[keep it simple, minimize resource requirements and just work](https://linkerd.io/design-principles/#), this means that
+most development should not even notice the presence of Linkerd and upkeep should be nigh non-existent.
 
-- Every service also gets an internal-only domain name that can be used by other
-  services to connect to it. The name is `<module-name>.<service-name>`. Requests
-  sent to this domain will automatically get load balanced b/w all healthy
-  containers.
+### Linkerd Viz
+For monitoring, Linkerd comes with an optional dashboard solution known as [Linkerd Viz](https://linkerd.io/2.11/features/dashboard/).
+It is not installed by default due to the non-negligible resource requirement (it installs Prometheus and Grafana
+as well as a few smaller tools), but can be easily added with the following commands: 
+
+```shell
+curl -fsL https://run.linkerd.io/install | sh   # Install linkerd CLI
+linkerd viz install | kubectl apply -f -        # Install linkerd monitoring stack
+linkerd viz dashboard                           # open the dashboard
+```
```

Comment:
this seems like a manual step, can we use helm for this? 
specially because we use helm for everything else.
https://artifacthub.io/packages/helm/linkerd2/linkerd-viz

this could also be packaged as a opta helm module.
but we would have to document how to open the dashboard without using linkerd cli.

Reply:
Yeah I'm just worried about linkerd being finicky about how the dashboard is installed-- you'll note how there's a real `linkerd viz dashboard` command for quick access and I just went for the straightforward, unfailing, answer.